Assignment 3.Rmd

```diff
@@ -26,16 +26,18 @@ The algorithm will treat each row as a value belonging to a person, so we need t
 
 ```{r}
 
-K2 <- 
+K2 <- K1[2:6]
 
 ```
 
 It is important to think about the meaning of missing values when clustering. We could treat them as having meaning or we could remove those people who have them. Neither option is ideal. What problems do you foresee if we recode or remove these values? Write your answers below:
 
+Since missing values are missing information they automatically reduce the explanatory power of our model. Recoding them assigns them values that we think they could mean but this may not be accurate. Removing them also inaccurately enhances the strength of the other non-missing values. It may skew our data when we remove the values.
```

Comment:
I would like you to think a bit more about how it may skew data in the case of k-means. k-means is about aggregating patterns, what might changing the way we treat a missing value do to how the algorithm performs?